paddle/framework/CMakeLists.txt

```diff
@@ -5,3 +5,4 @@ nv_test(dim_test SRCS dim_test.cu DEPS ddim)
 cc_test(variable_test SRCS variable_test.cc)
 cc_test(scope_test SRCS scope_test.cc)
 cc_test(enforce_test SRCS enforce_test.cc)
+proto_library(op_proto SRCS op_proto.proto attr_type.proto)
```

Comment:
I will give a unit test to demonstrate that. Actually, only `op_proto.proto` is needed for this `proto_library`.